client/components/App.jsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react'
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
-
+import { BrowserRouter as Router, Switch } from 'react-router-dom'
+import Dashboard from './Dashboard'
 import Header from './Header'
 import Register from './Register'
 
 const App = () => {
   return (
     <Router>
+      <Dashboard />
```

Comment:
Please write this similar to how you did Header and remove Header from this file. 
<Route path= component={} /> cheers!

Reply:
Remove the `<Dashboard/>` we just want the `<Route...` line for it.